src/main/java/team/reborn/energy/api/EnergyStorage.java

```diff
@@ -75,6 +76,11 @@ public interface EnergyStorage {
 	 */
 	EnergyStorage EMPTY = EnergyImpl.EMPTY;
 
+	/**
+	 * A {@link DataComponentType} for energy storage.
+	 */
+	DataComponentType<Long> ENERGY_COMPONENT = EnergyImpl.ENERGY_COMPONENT;
```

Comment:
I decided to put it here to cover the possibility of it being used on none items in the future. But maybe it would be best to move it to deter people from trying to directly query it. 👍